src/vtkh/rendering/CMakeLists.txt

```diff
@@ -35,45 +35,45 @@ set(vtkh_rendering_sources
   compositing/PartialCompositor.cpp
   )
 
-
-set(vtkh_rendering_deps vtkh_core vtkh_utils vtkh_filters)
-
-if(VTKm_CUDA_FOUND)
-    # triggers cuda compile
-    list(APPEND vtkh_rendering_deps cuda)
-endif()
-
-if(ENABLE_OPENMPJ)
-    list(APPEND vtkh_rendering_deps openmp)
-endif()
-
-blt_add_library(
-  NAME vtkh_rendering
-  SOURCES ${vtkh_rendering_sources}
-  HEADERS ${vtkh_rendering_headers}
-  DEPENDS_ON ${vtkh_rendering_deps}
-  )
-
-vtkm_add_target_information(vtkh_rendering DEVICE_SOURCES ${vtkh_rendering_sources})
-
-if(ENABLE_OPENMP)
-    blt_add_target_compile_flags(TO vtkh_rendering FLAGS "-DVTKH_USE_OPENMP")
+if (ENABLE_SERIAL)
+    set(vtkh_rendering_deps vtkh_core vtkh_utils vtkh_filters)
+    
+    if(VTKm_CUDA_FOUND)
+        # triggers cuda compile
+        list(APPEND vtkh_rendering_deps cuda)
+    endif()
+    
+    if(ENABLE_OPENMPJ)
```

Comment:
@danlipsa not your typo, but an easy one to fix -- can you fix as part of your pr so we don't miss it?
Should be:  `ENABLE_OPEMMP`

Reply:
Sure. I'll push a fix right now.